crates/hulk_behavior_simulator/src/bin/corner_kicks.rs

```diff
@@ -0,0 +1,62 @@
+use bevy::prelude::*;
+
+use scenario::scenario;
+use spl_network_messages::{GameState, PlayerNumber, SubState, Team};
+
+use hulk_behavior_simulator::{
+    game_controller::{GameController, GameControllerCommand},
+    robot::Robot,
+    time::{Ticks, TicksTime},
+};
+
+#[scenario]
+fn golden_goal(app: &mut App) {
```

Comment:
This name should match the scenario name. This function name is also used to generate the test.
Also check your other scenarios.

Reply:
Fixed on all scenarios